client/my-sites/customer-home/controller.jsx

```diff
@@ -48,7 +48,8 @@ export async function maybeRedirect( context, next ) {
 		// The new stepper launchpad onboarding flow isn't registered within the "page"
 		// client-side router, so page.redirect won't work. We need to use the
 		// traditional window.location Web API.
-		redirectToLaunchpad( slug, refetchedOptions?.site_intent );
+		const emailVerifiedParam = new URLSearchParams( window.location.search ).has( 'verified' );
```

Comment:
shouldn't it be `has.('emailVerified')` here

Reply:
The param is called `verified` on the homepage.

client/components/email-verification/index.js.
If you look at the first line in the `emailVerification` function, they are using `verified` as their query param.

Reply:
Oh I see, that makes sense

Reply:
`NB`: Did we consider using the `verified` query param instead of `emailVerified` when loading the launchpad page? It'd be nice to keep things consistent

Reply:
Yeah, I initially considered using that `verified`. Let me think about this for a moment and I will see if I update it.